src/megaapi_impl.cpp

```diff
@@ -6614,6 +6622,38 @@ bool MegaApiImpl::isScheduleNotifiable()
     }
 }
 
+int MegaApiImpl::abortPendingActions(error preverror)
+{
+    while (!backupsMap.empty())
+    {
+        std::map<int,MegaBackupController*>::iterator it = backupsMap.begin();
+        delete it->second;
+        backupsMap.erase(it);
+    }
+
+    while (!requestMap.empty())
+    {
+        std::map<int,MegaRequestPrivate*>::iterator it = requestMap.begin();
+        if(it->second)
```

Comment:
if it->second is null, it will loop forever?

Reply:
That's true at first sight, but no item in these maps will point to a `nullptr`, AFAIK. However, that makes me think about the reason for that checkup. I'll adjust the loop to avoid the situation. Thanks